### PR TITLE
add recipe for phykit 1.11.5

### PR DESCRIPTION
**Pull request by pauldg**
Add a recipe for the software tool Phytkit. PhyKIT is a UNIX shell toolkit for processing and analyzing phylogenomic data.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/phykit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+    - pk_pal2nal = phykit.phykit:thread_dna
+    - pk_p2n = phykit.phykit:thread_dna
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('cxx') }}
+  host:
```
[the remark this answers is not in the document]
Comment (pauldg) [reply recipes/phykit/meta.yaml:127]: Thank you for looking into it, it seems to have resolved the issue!

File: recipes/phykit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+    - pk_create_concatenation_matrix --help
+    - pk_create_concat --help
+    - pk_cc --help
+    - pk_thread_dna --help
+    - pk_pal2nal --help
+    - pk_p2n --help
+
+about:
+  home: "https://github.com/jlsteenwyk/phykit"
+  license: UNKNOWN
```
Comment (marcelm) [inline recipes/phykit/meta.yaml:256]: This should be MIT according to what it says in your repo, correct?

File: recipes/phykit/meta.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcelm) [reply recipes/phykit/meta.yaml:127]: The shorter the recipe gets, the fewer problems you have :-)

File: recipes/phykit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+  license_family: OTHER
+  license_file:
```
Comment (marcelm) [inline recipes/phykit/meta.yaml:258]: You should remove the unused fields: `license_family` (that is rarely set in Bioconda recipes anyway), `license_file`, `doc_url`, `dev_url`.

File: recipes/phykit/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+    - pk_p2n = phykit.phykit:thread_dna
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - biopython >=1.79
+    # - cython
```
Comment (marcelm) [inline recipes/phykit/meta.yaml:132]: Please remove this commented line (just to have it look a little cleaner)

File: recipes/phykit/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+    - numpy >=1.18.2
+    - python
```
Comment (marcelm) [inline recipes/phykit/meta.yaml:134]: Is the program compatible with Python 2.7? If not, you should write this as `python >=3`.

File: recipes/phykit/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+  license: UNKNOWN
+  license_family: OTHER
+  license_file: 
+  summary: "PhyKIT is a UNIX shell toolkit for processing and analyzing phylogenomic data."
+  doc_url: 
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - JLSteenwyk
```
Comment (marcelm) [inline recipes/phykit/meta.yaml:265]: The recipe maintainer is the person who takes care of the Bioconda recipe. If you want to add anyone other than yourself, make sure you have their permission.

File: recipes/phykit/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,265 @@
...
+    - pk_treeness_over_rcv = phykit.phykit:treeness_over_rcv
+    - pk_toverr = phykit.phykit:treeness_over_rcv
+    - pk_tor = phykit.phykit:treeness_over_rcv
+    - pk_create_concatenation_matrix = phykit.phykit:create_concatenation_matrix
+    - pk_create_concat = phykit.phykit:create_concatenation_matrix
+    - pk_cc = phykit.phykit:create_concatenation_matrix
+    - pk_thread_dna = phykit.phykit:thread_dna
+    - pk_pal2nal = phykit.phykit:thread_dna
+    - pk_p2n = phykit.phykit:thread_dna
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (marcelm) [inline recipes/phykit/meta.yaml:124]: The standard incantation is:
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
```

File: recipes/phykit/meta.yaml (hunk 2 above)
Comment (pauldg) [reply recipes/phykit/meta.yaml]: That's correct, I've changed it

File: recipes/phykit/meta.yaml (hunk 6 above)
Comment (pauldg) [reply recipes/phykit/meta.yaml]: The developer of the package is aware that I will be maintaining the recipe. I've changed it to my github username.

**Comment by pauldg** [comment]
@BiocondaBot please add label

**Review by marcelm — APPROVED** [review]

**Comment by marcelm** [comment]
@biocondabot please merge

**Pull request merged**